src/main/java/edu/greenblitz/robotName/commands/shooter/Shoot.java

```diff
@@ -0,0 +1,21 @@
+package edu.greenblitz.robotName.commands.shooter;
+
+import edu.greenblitz.robotName.commands.shooter.flyWheel.StopFlyWheel;
+import edu.greenblitz.robotName.commands.shooter.funnel.RunFunnelWithSafetyRotations;
+import edu.greenblitz.robotName.subsystems.shooter.FlyWheel.FlyWheelConstants;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class Shoot extends SequentialCommandGroup {
+
+    public Shoot() {
```

Comment:
why isnt it shoioting? the flywheel is not spinning

Reply:
the flywheel is spinning while moving to the shooting position